packages/algolia/schema/gp2/algolia-reverse-timestamp-schema.json

```diff
@@ -9,14 +9,14 @@
     "authors.id",
     "documentType",
     "eventTypes",
-    "project.id",
+    "projects.id",
     "projectIds",
     "region",
     "speakers.user.id",
     "status",
     "type",
     "tagIds",
-    "workingGroup.id",
+    "workingGroups.id",
```

Comment:
i think some elements (maybe events?) are still using a workingGroup property - are we not filtering them by this attribute anywhere? if we are you are going to need to have both of them here

Reply:
actually we are not using WG to filter events